manta-pay/src/crypto/ecc/arkworks.rs

```diff
@@ -408,49 +410,41 @@ where
 
 macro_rules! impl_processed_scalar_mul {
     ($curve: ty) => {
-impl<CV>
-    ecc::PreprocessedScalarMul<
-        Compiler<$curve>,
+        impl<CV>
+            ecc::PreprocessedScalarMul<
+                Compiler<$curve>,
+                {
+                    <<$curve as ProjectiveCurve>::ScalarField as PrimeField>::Params::MODULUS_BITS
+                        as usize
+                },
```

Comment:
Can you try to make an alias for this? Not sure if it'll work because it may involve an unstable feature, but worth a try:

```rust
type ScalarParams<C> = <<C as ProjectiveCurve>::ScalarField as PrimeField>::Params;
```

Then you can call it as `{ ScalarParams::<$curve>::MODULUS_BITS }`. Also, we can call this from the test.

Reply:
it works! done